src/app/src/components/PageShell.tsx

```diff
@@ -67,7 +68,20 @@ const lightTheme = createAppTheme(false);
 const darkTheme = createAppTheme(true);
 
 function Layout({ children }: { children: React.ReactNode }) {
-  return <div>{children}</div>;
+  const location = useLocation();
+  const isConfigPage = location.pathname.includes('/setup');
```

Comment:
Would prefer a more general solution than this. Can we change the styles on the redteam setup ui instead?